RIOT-OS/RIOT
# gnrc/ipv6_auto_subnets: improve documentation about subnet generation and RIO

### Contribution description

This better illustrates how subnet generation works and why we need the *Route Information Option* in a Router Advertisement to the upstream network for this to properly work.


### Testing procedure

Read [the documentation](https://output.circle-artifacts.com/output/job/1c5280c0-c298-4d75-82f4-6e8c926b70ed/artifacts/0/doc/group__net__gnrc__ipv6__auto__subnets.html) generated by CircleCI and see if things make sense.


### Issues/PRs references
